### PR TITLE
Generate enums in samples

### PR DESCRIPTION
**Pull request by pakrym-stripe**
Use enums in generated samples.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .env
 *.test
 *.coverprofile
+/.idea
```
Comment (yejia-stripe) [inline .gitignore:5]: What's this?

**Review by yejia-stripe — COMMENTED** [review]
Lgtm!

File: .gitignore (hunk 1 above)
Comment (pakrym-stripe) [reply .gitignore:5]: Directory that https://www.jetbrains.com/go/ uses for its temporary files.

**Review by yejia-stripe — APPROVED** [review]
Sorry thought I approved this

**Pull request merged**